apache-rat-core/src/main/java/org/apache/rat/report/xml/writer/impl/base/XmlWriter.java

```diff
@@ -627,6 +627,39 @@ public IXmlWriter closeElement() throws IOException {
         return this;
     }
 
+    /**
+     * Closes back to and including the last instance of the specified element nema.
+     * @param name The name of the element to close.
+     * @return this object
+     * @throws OperationNotAllowedException if called before any call to
+     * {@link #openElement} or after the first element has been closed
+     */
+    @Override
+    public IXmlWriter closeElement(CharSequence name) throws IOException {
```

Comment:
Is NPE-safeness an issue here? Should we throw an exception if invalid/null is passed in here?